### PR TITLE
don't use docker-machine for k8petstore on linux

### LINKED ISSUE #17063: dont use docker-machine for k8petstore on linux
no need for docker machine  tooling for people hacking on k8petstore on a linux box.
easy patch , one liner, will submit shortly

### PR DESCRIPTION
**Pull request by jeffvance**
Fixes #17063

### CODE REVIEW

File: examples/k8petstore/docker-machine-dev.sh [hunk 1]
```diff
@@ -16,15 +16,20 @@
 
 !/bin/bash
```
Comment (zmerlynn) [inline examples/k8petstore/docker-machine-dev.sh]: What is this line?

**Comment by zmerlynn** [comment]
@jeffvance: BTW, if you talk to @smarterclayton, there's a way to get @googlebot to stop hassling you as a RedHatter (which you clearly are).

**Comment by zmerlynn** [comment]
@k8s-bot ok to test

**Comment by jayunit100** [comment]
Looks good.  

I also suggested to jeff that we need to stop building redis from scratch...  Rebuilding redis containers against a specific OS is unnecessary.

@jeffvance i think you can also fix the `!/bin/bash` easily in this PR.

Thanks!

File: examples/k8petstore/docker-machine-dev.sh (hunk 1 above)
Comment (jeffvance) [inline examples/k8petstore/docker-machine-dev.sh]: Yes, fixed that in next fix

**Comment by jayunit100** [comment]
I've reviewed and tested this... This looks good to me.

Maybe we can get a quick LGTM from the commiters?

**Comment by jeffvance** [comment]
@googlebot I signed it!

**Comment by jeffvance** [comment]
@googlebot I signed it !

**Pull request merged**